File: dbflow/src/androidTest/java/com/raizlabs/android/dbflow/test/structure/caching/CacheableModel2.java
package com.raizlabs.android.dbflow.test.structure.caching;

import com.raizlabs.android.dbflow.annotation.Column;
import com.raizlabs.android.dbflow.annotation.PrimaryKey;
import com.raizlabs.android.dbflow.annotation.Table;
import com.raizlabs.android.dbflow.structure.BaseModel;
import com.raizlabs.android.dbflow.test.TestDatabase;

@Table(database = TestDatabase.class, cachingEnabled = true)
public class CacheableModel2 extends BaseModel {

    @Column
    @PrimaryKey
    int id;
}

File: dbflow-tests/src/test/java/com/raizlabs/android/dbflow/structure/caching/CacheableModel2.kt
package com.raizlabs.android.dbflow.structure.caching

import com.raizlabs.android.dbflow.annotation.Column
import com.raizlabs.android.dbflow.annotation.PrimaryKey
import com.raizlabs.android.dbflow.annotation.Table
import com.raizlabs.android.dbflow.structure.BaseModel
import com.raizlabs.android.dbflow.TestDatabase

@Table(database = TestDatabase::class, cachingEnabled = true)
class CacheableModel2 : BaseModel() {

    @Column
    @PrimaryKey
    var id: Int = 0
}
